lineman/app/models/base_endpoints_interface.coffee

```diff
@@ -0,0 +1,22 @@
+angular.module('loomioApp').factory 'BaseEndpointsInterface', (RestfulClient) ->
```

Comment:
I'm not 100% on it either, but basically I know that

- We have use cases here where we want to interact with the API, but not
necessarily deal with the record store
- The only way to access the RestfulClient at the moment is either directly
(ick), or through the BaseRecordsInterface, which requires a model class to
make it work.
(I'm looking at the line `@restfulClient = new
RestfulClient(@model.plural)` in the BaseRecordsInterface)

It felt pretty good to write Records.invitables.fetch() and
Records.invitations.create(), but maybe it wants to be even more separate,
like Endpoints.invitables.fetch() and Endpoints.invitations.create()

??